hbase-common/src/main/java/org/apache/hadoop/hbase/ScheduledChore.java

```diff
@@ -183,18 +176,21 @@ public void run() {
       if (LOG.isInfoEnabled()) LOG.info("Chore: " + getName() + " was stopped");
     } else {
       try {
+        // TODO: Histogram metrics per chore name.
+        // For now, just measure and log if DEBUG level logging is enabled.
+        long start = 0;
+        if (LOG.isDebugEnabled()) {
+          start = System.nanoTime();
+        }
         if (!initialChoreComplete) {
           initialChoreComplete = initialChore();
         } else {
-          timeMeasurement.measure(() -> {
             chore();
-            return null;
-          });
-          if (LOG.isInfoEnabled() && (System.nanoTime() - lastLog > FIVE_MINUTES_IN_NANOS)) {
```

Comment:
Do we want some duration threshold that triggers escalating the log level to info (or warn), as was done before?

Reply:
Not opposed to the idea at all, but it's going to depend on the chore, right?

Reply:
yeah i think you're right. let's leave it for now.